src/explore-education-statistics-common/src/components/form/rhf/RHFForm.tsx

```diff
@@ -118,12 +119,10 @@ export default function RHFForm<TFormValues extends FieldValues>({
         try {
           await onSubmit(values as TFormValues);
         } catch (error) {
-          if (!error) {
-            return;
-          }
+          if (showSubmitError && isErrorLike(error)) {
```

Comment:
Do we want to log it even if we choose not to show it actually?

Reply:
No, because it gets re-thrown. This will typically get handled by the global error handler, but it's really up to the consuming code to decide what to do with such an error.